src/app/app.component.spec.ts

```diff
@@ -1,23 +1,23 @@
-/* tslint:disable:no-unused-variable */
```

Comment:
you can probably take the master version when merging in as it might be using the new spec